modules/search/class-jetpack-search-customize.php

```diff
@@ -117,6 +117,24 @@ public function customize_register( $wp_customize ) {
 			)
 		);
 
+		$id = $setting_prefix . 'overlay_transition';
+		$wp_customize->add_setting(
+			$id,
+			array(
+				'default'   => true,
```

Comment:
```suggestion
				'default'   => false,
```

Reply:
I actually wanted it to default to `true`, but the class wasn't being removed when I checked the option off on the Customizer. This was the only way it worked for me, but maybe I did something wrong in my testing?